docs/reference/components/sources/nats.cue

```diff
@@ -0,0 +1,75 @@
+package metadata
+
+components: sources: nats: {
+	title: "NATS"
+
+	features: {
+		collect: {
+			checkpoint: enabled: false
+			tls: {
+				enabled:                true
```

Comment:
It seems like this should be `false` no?

Reply:
Should I get rid of the tls object entirely? I'm getting a cue validation error when I change it to false.

Reply:
And yes, to clarify, I haven't implemented TLS support.

Reply:
Ah, weird, I thought you could just do `tls: enabled: false`, but I believe the default is false anyway so this is fine.